src/extensions/storage_macro/chunk.rs

```diff
@@ -36,33 +22,62 @@ impl crate::application::CoLink {
                     &payload[offset..offset + chunk_size],
                 )
                 .await?;
-            chunk_paths.push(response);
+            chunk_paths.push(response.split('@').last().unwrap().to_string()); // only store the timestamps
             offset += chunk_size;
             chunk_id += 1;
         }
+        Ok(chunk_paths)
+    }
+
+    fn _check_chunk_paths_size(&self, chunk_paths: Vec<String>) -> Result<String, Error> {
+        let chunk_paths_string = chunk_paths.join(";");
+        if chunk_paths_string.len() > CHUNK_SIZE {
+            return Err(format!(
+                "File too large: failed to store {} chunks references in metadata",
+                chunk_paths.len()
+            )
+            .into());
+        }
+        Ok(chunk_paths_string)
+    }
+
+    #[async_recursion]
+    pub(crate) async fn _create_entry_chunk(
+        &self,
+        key_name: &str,
+        payload: &[u8],
+    ) -> Result<String, Error> {
+        let metadata_key = format!("{}:chunk_metadata", key_name);
+
```

Comment:
again, do not use too many empty lines

Reply:
please also check other files that you committed, only use empty lines when the separation helps with the understanding